EIPS/eip-1538.md

```diff
@@ -3,12 +3,13 @@ eip: 1538
 title: Transparent Contract Standard
 author: Nick Mudge <nick@mokens.io>
```

Comment:
Both of these EIPs are authored by Nick Mudge, but the email address is different.  Did you just change email addresses?  If so, can you send a quick email to me (micah at zoltu dot net) from `nick@mokens.io` letting me know that this is you or can one of the other editors verify?

I generally prefer GitHub accounts as authors for this reason, though I can appreciate that email is more censorship resistant so I generally support people using email as well.

Reply:
It is the same person. I will send you an email. What is your email address?

I will change nick@mokens.io to nick@perfectabstractions.com, so the email address is consistent.

Reply:
I changed the email address.